src/server/src/Server.cpp

```diff
@@ -77,36 +82,36 @@ bool Server::configure(ResourceFinder& rf)
         return false;
     }
 
-    if (!drdIsInitialized () && drdAutoInit () < 0)
+    if (!drdIsInitialized() && drdAutoInit() < 0)
     {
         std::cout << "Error: cannot initialize the Omega.3 robot." << std::endl;
 
         return false;
     }
-    else if (drdStart () < 0)
+    else if (drdStart() < 0)
     {
         std::cout << "Error: cannot start the Omega.3 robot." << std::endl;
 
         return false;
     }
 
     /* Move in a safe position and idle the robot. */
+
     enable_position_control();
-    drdMoveToPos(0.0, 0.0, 0.0, false);
-    state_ = State::PositionControl;
+    drdMoveToPos(0.0, 0.0, 0.0, false); // stay at initial position
+
+    state_ = State::Idle;
```

Comment:
Can you please explain the reason of the change from `state_ = State::PositionControl` to `Idle`?

Reply:
I can't remember I have done it. But maybe because I needed to move it freedly? Went back to PositionControl

Reply:
Thank you. However you left the comment `// stay at initial position` there. Can you remove it?

Also it seems that you added some spaces with respect to the original code. Can you please have a look at it [here](https://github.com/robotology-playground/yarp-omega3/blob/470f3ce778b3a9f9b73d91ef24ae9c01ec597e9b/src/server/src/Server.cpp#L93)?